gobblin-runtime/src/main/java/org/apache/gobblin/service/monitoring/FlowStatusGenerator.java

```diff
@@ -176,6 +182,9 @@ private boolean isJobRunning(JobStatus jobStatus) {
    * Send kill request for the given flow
    */
   public void killFlow(String flowGroup, String flowName, Long flowExecutionId) {
```

Comment:
Quick question: Why is there a killFlow() method in FlowStatusGenerator? Can this method be relocated into FlowExecutionResource which seems like the right location? Looks like you only want the eventBus instance in FlowExecutionResource, which can be injected the same way that FlowStatusGenerator is.